ui/src/components/building-block/OurValues.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Box, Heading, Text, Stack, Image } from '@chakra-ui/react';
+
+const values = [
+  {
+    title: 'Direction',
+    description:
+      "Children to come to a saving knowledge of Jesus Christ and be rooted in God's Word and the gospel.",
+    icon: '/images/building-blocks/DirectionIcon.png',
+    bg: '#DBF1F1',
+    shadow: '0.625rem 0.625rem 0 #77C2C6',
+  },
+  {
+    title: 'Delight',
+    description:
+      'Children to experience the joy of being in an intimate relationship with Jesus and the church community.',
+    icon: '/images/building-blocks/DelightIcon.png',
+    bg: '#F5F0E0',
+    shadow: '0.5rem 0.5rem 0 #FFE29A',
+  },
+  {
+    title: 'Destiny',
+    description:
+      "Children to grow as a disciple of Jesus, to daily live out God's call in their lives, and raise up future disciples of Christ.",
+    icon: '/images/building-blocks/DestinyIcon.png',
+    bg: '#F1E6E6',
+    shadow: '0.5rem 0.5rem 0 #F7B2B7',
+  },
+];
+
+const OurValues = () => (
+  <Box maxW="90rem" p={{ base: '1rem', md: '2rem' }} mb={20}>
```

Comment:
consider using <Container> with maxW="container.xl" to standardize the gutter width between all sections

Reply:
maxW="90rem" sounds incorrect xD

Reply:
ok thanks